pkgs/by-name/go/gomuks-web/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo123Module,
+  buildNpmPackage,
+  stdenv,
+  olm,
+  withGoolm ? false,
+  ...
+}:
+
+let
+  cppStdLib = if stdenv.hostPlatform.isDarwin then "-lc++" else "-lstdc++";
+
+in
+buildGo123Module rec {
+  pname = "gomuks-web";
+  version = "0.4.0-unstable-2025-04-22";
+  rev = "fd257ed74c9df42e5b6d14d3c6a283f557f61666";
+  sha256 = "sha256-jMDLfiwkUme2bxE+ZEtUoNMwZ7GuGGzCV2dH1V87YtQ=";
+
+  src = fetchFromGitHub {
+    owner = "tulir";
+    repo = "gomuks";
+    inherit rev sha256;
+  };
+
+  frontend = buildNpmPackage {
+    name = "${pname}_${version}-frontend";
+    src = "${src}/web";
+    inherit version;
+
+    npmBuildScript = "build";
+    npmDepsHash = "sha256-Mt2gJ1lLT3oQ3RKr3XTVFXkS/Xmjy0gahbdaxxrO+6g=";
+
+    installPhase = ''
+      cp -r dist $out
+    '';
+  };
+
+  vendorHash = "sha256-qeSxxd9ml2ENAYSPkdd1OWqy2DULnwLUVkKje47uT/I=";
+
+  buildInputs = (lib.optional (!withGoolm) olm) ++ (lib.optional withGoolm stdenv.cc.cc.lib);
+
+  CGO_LDFLAGS = lib.optional withGoolm [ cppStdLib ];
```

```suggestion
  CGO_LDFLAGS = lib.optional withGoolm cppStdLib;
```

or:
```suggestion
  CGO_LDFLAGS = lib.optionals withGoolm [ cppStdLib ];
```

Current: builds list of lists, probably not what you want.